escrow/contracts/escrow.sol

```diff
@@ -0,0 +1,91 @@
+// SPDX-License-Identifier: Unlicensed
+pragma solidity ^0.8.9;
+
+import "@acala-network/contracts/token/Token.sol";
+
+contract Escrow {
+    address payable public requestor;
+    address payable public serviceProvider;
```

Comment:
I think that `serviceProvider` could be renamed into `beneficiary` to be in line with the current tutorials.

Same question as above about this being `payable`.